templates/careers/list.html

```diff
@@ -0,0 +1,71 @@
+        <div
+          class="absolute top-1 md:top-2 left-1 md:left-2 border-t border-l border-purple-100 h-10 md:h-20 w-10 md:w-20">
+          <div
+            class="absolute top-1 md:top-2 left-1 md:left-2 border-t border-l border-purple-200 h-10 md:h-20 w-10 md:w-20">
+            <div
+              class="absolute top-1 md:top-2 left-1 md:left-2 border-t border-l border-purple-300 h-10 md:h-20 w-10 md:w-20">
+              <div
+                class="absolute top-1 md:top-2 left-1 md:left-2 border-t border-l border-purple-400 h-10 md:h-20 w-10 md:w-20">
+                <div
+                  class="absolute top-1 md:top-2 left-1 md:left-2 border-t border-l border-purple-500 h-10 md:h-20 w-10 md:w-20">
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="absolute top-0 right-0 border-t border-r border-purple-50 h-10 md:h-20 w-10 md:w-20">
+        <div
+          class="absolute top-1 md:top-2 right-1 md:right-2 border-t border-r border-purple-100 h-10 md:h-20 w-10 md:w-20">
+          <div
+            class="absolute top-1 md:top-2 right-1 md:right-2 border-t border-r border-purple-200 h-10 md:h-20 w-10 md:w-20">
+            <div
+              class="absolute top-1 md:top-2 right-1 md:right-2 border-t border-r border-purple-300 h-10 md:h-20 w-10 md:w-20">
+              <div
+                class="absolute top-1 md:top-2 right-1 md:right-2 border-t border-r border-purple-400 h-10 md:h-20 w-10 md:w-20">
+                <div
+                  class="absolute top-1 md:top-2 right-1 md:right-2 border-t border-r border-purple-500 h-10 md:h-20 w-10 md:w-20">
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="text-center pt-12 md:pt-20 px-12 sm:px-16 md:px-24 lg:px-36 ">
+    <h1 class="text-2xl-5xl">Careers</h1>
+  </div>
+  <div class="mt-16 md:mt-24">
+    <div class="xl:max-w-screen-xl mx-auto md:px-12 lg:px-16">
+      <div class="bg-gray-800 shadow-lg shadow-xl shadow-md shadow-sm px-6 sm:px-12 md:px-12 lg:px-16 py-16 md:py-20">
+        <h2 class="text-gray-50 font-normal text-3xl-4xl text-center">Working at the Haskell Foundation</h2>
+        <p class="mt-10 lg:text-xl text-gray-300 leading-relaxed text-center">
+            Interested in working at the crossroads of the Haskell ecosystem?
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="mt-16 md:mt-24">
+    <div class="text-2xl-5xl text-center md:px-12">Open</div>
+    <div class="max-w-screen-xl mx-auto grid gap-8 lg:grid-cols-2 md:px-12">
+      $for(openreqs)$
+      $partial("templates/careers/tile.html")$
+      $endfor$
+    </div>
+
+    <div class="text-2xl-5xl text-center md:px-12">Closed</div>
+    <div class="max-w-screen-xl mx-auto grid gap-8 lg:grid-cols-2 md:px-12">
+      $for(closedreqs)$
```

Comment:
Is the intent to have a graveyard of old job postings here? That seems unnecessary to me. But maybe I've misunderstood the section...